tripy/tripy/backend/mlir/compiler.py

```diff
@@ -54,18 +54,20 @@ def __init__(self, trt_builder_opt_level=0) -> None:
         self.trt_builder_opt_level = trt_builder_opt_level
 
     def _make_mlir_opts(self, trt_builder_opt_level):
-        opts = compiler.StableHLOToExecutableOptions(self.compiler_client,
-                                                     [f"--tensorrt-timing-cache-path={G_TIMING_CACHE_FILE}",
-                                                      f"--tensorrt-builder-opt-level={trt_builder_opt_level}",
-                                                      "--tensorrt-strongly-typed=True"])
+        opts = [
+            f"--tensorrt-timing-cache-path={G_TIMING_CACHE_FILE}",
+            f"--tensorrt-builder-opt-level={trt_builder_opt_level}",
+            "--tensorrt-strongly-typed=True",
+        ]
         if config.enable_mlir_debug or config.enable_tensorrt_debug:
-            opts.set_debug_options(
-                config.enable_mlir_debug,
-                config.mlir_debug_types if config.enable_mlir_debug else [],
-                config.mlir_debug_tree_path if config.enable_mlir_debug else None,
-                config.tensorrt_debug_path if config.enable_tensorrt_debug else None,
-            )
-        return opts
+            opts.append("--debug=true")
```

Comment:
Why are we always enabling debug flag? What does debug=True mean?

Reply:
Not always, only if either `TRIPY_MLIR_DEBUG_ENABLED` or `TRIPY_TRT_DEBUG_ENABLED` is set.

Reply:
Ah, missed the unchanged if condition. What does debug=True do in `mlir-tensorrt`? There is already debug-only flag that should have indicated `debug=True`.

Reply:
See this: https://github.com/NVIDIA/TensorRT-Incubator/blob/main/mlir-tensorrt/compiler/include/mlir-tensorrt/Compiler/Options.h#L53
I want used https://github.com/NVIDIA/TensorRT-Incubator/blob/main/mlir-tensorrt/test/python/mlir_tensorrt_compiler/compiler_api/test_compiler_debug_dump.py as a reference.